src/Billing/Claim.php

```diff
@@ -25,27 +25,28 @@ class Claim
     public const X12_VERSION = '005010X222A1';
     public const NOC_CODES = array('J3301'); // special handling for not otherwise classified HCPCS/CPT, not many so can add more here
 
-    public $pid;               // patient id
-    public $encounter_id;      // encounter id
-    public $procs;             // array of procedure rows from billing table
-    public $diags;             // array of icd codes from billing table
-    public $diagtype = "ICD10"; // diagnosis code_type; safe to assume ICD10 now
+    public int $pid;               // patient id
+    public int $encounter_id;      // encounter id
+    public string $procs;             // array of procedure rows from billing table
+    public string $diags;             // array of icd codes from billing table
+    public string $diagtype = "ICD10"; // diagnosis code_type; safe to assume ICD10 now
```

Comment:
These are properties and PHP7.4 doesn't support. Or is it 7.3 lol I can't keep up.
Please check!

Reply:
also need to be careful since pretty sure will break if any are set to null (ie. not set). for the fix would not include the property changes since will be going back to 6.1.0 .

Reply:
here's the [doc to setup phpstorm for openemr codesniffing standards](https://www.open-emr.org/wiki/index.php/OpenEMR_PHPStorm_Setup)

to reiterate, only have to fix these errors:
![Screenshot from 2022-03-27 19-44-10](https://user-images.githubusercontent.com/5618372/160306418-5c8c777a-026d-4fe1-92ae-f08f19819e1b.png)

thanks @growlingflea!

Reply:
that's because you have your environment set up for php8.

Reply:
oh sorry, yes just fix styling errors @growlingflea

Reply:
Ok, repushing.